class DataProcessingError(Exception):
    pass

class ValidationError(Exception):
    pass